biosys/apps/main/models.py

```diff
@@ -479,15 +479,28 @@ class Meta:
         ordering = ['id']
 
 
+def media_path(instance, filename):
```

Comment:
is this the right place for this function? Also, maybe call it ```get_media_path()```

Reply:
Well it is used only in the Media model for the field `file`.
```
class Media(models.Model):
    file = models.FileField(upload_to=media_path)
```
This method should not be used.
I've added a comment
I've tried to put it as a static method inside the model but it didn't work.
https://docs.djangoproject.com/en/1.11/ref/models/fields/#filefield

I will change the name